modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -2070,4 +2071,114 @@ describe('V2 Wallet:', function () {
       });
     });
   });
+
+  describe('unspents APIs with string type minValue and maxValue', function () {
+    // using the same test for fanout and consolidate unspent API
+    it('consolidateUnspents should handle string type minValue and maxValue', async function () {
+      const params = { minValue: '123', maxValue: '9000000000000000000123' };
```

Comment:
How did you arrive at that number?

Reply:
I was just using very large number that doesn't fit into number. Changed it to 10000000000000123